app/controllers/clicked_controller.rb

```diff
@@ -23,6 +23,7 @@ def click_class
   end
 
   def permitted_params
+    params.delete(:clicked)
```

Comment:
See my question in SRCH-4881.  In a perfect world I'd like not to need this, but I can't figure out a workaround.  Input welcome.

Reply:
Hi @stevenbarragan @luismetzger @krbhavith - any thoughts on this?

Reply:
It's hard to follow where this is coming from. From the logs from [here](https://cm-jira.usa.gov/browse/SRCH-4881?focusedId=1850760&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1850760)
```
16:57:07 web.1  |   Parameters: {"affiliate"=>"test_affiliate", "url"=>"https://search.gov", "module_code"=>"BOOS", "position"=>1, "query"=>"best bet", "vertical"=>"web", "clicked"=>{"affiliate"=>"test_affiliate", "url"=>"https://search.gov", "module_code"=>"BOOS", "position"=>1, "query"=>"best bet", "vertical"=>"web"}}
16:57:07 web.1  | Unpermitted parameter: :clicked. Context: { controller: ClickedController, action: create, request: #<ActionDispatch::Request:0x00007f89c83cc950>, params: {"affiliate"=>"test_affiliate", "url"=>"https://search.gov", "module_code"=>"BOOS", "position"=>1, "query"=>"best bet", "vertical"=>"web", "controller"=>"clicked", "action"=>"create", "clicked"=>{"affiliate"=>"test_affiliate", "url"=>"https://search.gov", "module_code"=>"BOOS", "position"=>1, "query"=>"best bet", "vertical"=>"web"}} }
```

I can only tell that the params are been duplicated on the `clicked` key. Are the logs from [this request](https://github.com/GSA/search-gov/pull/1488/files#diff-6627eb1616884db52ea1b6eeac2a9cd658aceec88de7cfaeef31a97223ba0403R81)? Where does `params["clicked"]` gets populated?